src/utils/cpm.js

```diff
@@ -4,7 +4,10 @@ import {logError} from '../utils.js';
 
 export function adjustCpm(cpm, bidResponse, bidRequest, {index = auctionManager.index, bs = bidderSettings} = {}) {
   bidRequest = bidRequest || index.getBidRequest(bidResponse);
-  const bidCpmAdjustment = bs.get(bidResponse?.bidderCode || bidRequest?.bidder, 'bidCpmAdjustment');
+
+  const adjustForAlternateBids = bs.get(bidResponse?.adapterCode, 'adjustForAlternateBids');
```

Comment:
`adjustFor` reads weird; we're adjusting the bids, not adjusting _for_ the bids.

Reply:
changed the var name to be just `adjustAlternateBids`